components/brave_wallet/browser/eth_tx_controller.cc

```diff
@@ -19,9 +19,99 @@
 #include "brave/components/brave_wallet/browser/eth_data_builder.h"
 #include "brave/components/brave_wallet/browser/eth_json_rpc_controller.h"
 #include "brave/components/brave_wallet/browser/keyring_controller.h"
+#include "components/grit/brave_components_strings.h"
+#include "ui/base/l10n/l10n_util.h"
 
 namespace brave_wallet {
 
+// static
+void EthTxController::FixMissingTxData(mojom::TxDataPtr* tx_data) {
```

Comment:
shouldn't we use default value in mojom instead of fixing up here?
```
struct TxData {
  string nonce;
  string gas_price;
  string gas_limit;
  string to;
  string value = "0x00";
  array<uint8> data;
};
```

Reply:
That would not be the same effect because for example callers can use `TxData::New(...args)` and if they specify an empty `value`, it will not be filled with `"0x00"`.  That default value would only be used if a caller did `TxData::New()` and then filled each member it could.  But we cannot say for sure all people will do that even if we change the current code to not use `TxData::New(...args)`.

Reply:
Turns out I can just remove `FixMissingTransactionData` and no default value is needed. Fixed in 6b070e62386da740af68c1fa981fb3a8fca61758